testing/internal/e2e/boundary/authtoken.go

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package boundary
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/boundary/api/authtokens"
+	"github.com/hashicorp/boundary/testing/internal/e2e"
+	"github.com/stretchr/testify/require"
+)
+
+// GetAuthenticationTokenIdCli uses the CLI to get an auth-token ID by its name
```

Comment:
I think I was confused by this comment. To me, this reads as if you're looking up an auth-token ID using its name, but based on our earlier conversation, this is using the token name to authenticate the command.

Two things:
- How do we know we're retrieving the desired auth token here? The test is assuming this is the user's auth token? Is that the only one that is in the list?
- To keep things methods in this package consistent, should we not make `tokenName` required?

Reply:
After discussion with @moduli, we decided to keep current approach and rename the method for more clarity.
Also, I've updated the comment to provide more detailed description of what the method does